Dockerfile

```diff
@@ -20,8 +20,9 @@ ARG HTTPS_PROXY
 ARG HTTP_PROXY
 ARG UI
 ARG SWAGGER
+ARG LDFLAGS
 
 COPY . /src
 WORKDIR /src
 RUN --mount=type=cache,target=/root/.cache/go-build \
-    make binary
+    IMG_LDFLAGS=$LDFLAGS make binary
```

Comment:
Why not name "IMG_LDFLAGS" as "LDFLAGS" directly?

Reply:
We need to reference `IMG_LDFLAGS` to `LDFLAGS` in Makefile, If we use `LDFLAGS` directly, an error will be report when compiling: 

```
 Recursive variable `LDFLAGS' references itself (eventually).  Stop.
```